src-test/copyright.c

```
@@ -0,0 +1,103 @@
+	}
+	out[i] = 0;
+}
+
+static bool print_copyrights_item(uint8_t *key, uint8_t *subkey, int subkey_ln, uint8_t *data, uint32_t datalen, int iteration, void *ptr)
+{
+	match_data *match = ptr;
+
+	decrypt_data(data, datalen, "copyright", key, subkey);
+
+	char *CSV = calloc(datalen + 1, 1);
+	memcpy(CSV, (char *) data, datalen);
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)